src/OpenBinary/reader.jl

```diff
@@ -453,29 +448,123 @@ function PSRI.goto(graf::Reader, t::Integer, s::Integer = 1, b::Integer = 1)
     return nothing
 end
 
-function _get_position(graf, t::Integer, s::Integer, b::Integer)
-    if PSRI.is_hourly(graf)
+function _get_relative_offset(ptr::Reader)
+    return 4 * ptr.agents_total * ptr.scenario_total * (ptr.first_relative_stage - 1)
+end
+
+function _get_relative_offset(::Any)
```

Comment:
This is necessary?

Reply:
Yes. The `Writer` also uses `_get_position`.

Reply:
And, yes, it is not aware of the relative offset yet. If you read a 0-index graf and then attempt to write it, it will get 1-indexed.

Part II - Enhance OpenBinary Reader is coming soon.